contents.json

```diff
@@ -16466,6 +16466,26 @@
       ],
       "date_added": "May 28 2018",
       "suggested_by": "@txaiwieser"
-   }
-   ]
+     },
+     {
+       "title": "Mojilist",
+       "description": "🛍 Create shopping lists with emojis!",
```

Comment:
Sorry this is not in the rules but pls no emoji in descriptions

Reply:
Shure! I'm going to fix this in the evening.